skfp/fingerprints/atom_pair.py

```diff
@@ -63,6 +64,11 @@ class AtomPairFingerprint(FingerprintTransformer):
     count : bool, default=False
         Whether to return binary (bit) features, or their counts.
 
+    normalize: bool, default=False
+        Wheather to scale count fingerprint by the heavy atom count (HAC) to
+        obtain a proportionality to molecular size. Vector values are
```

Comment:
"to molecule size", I think, since we are talking about a particular molecule, not "molecular" in general